hydra-task/src/main/java/com/addthis/hydra/task/map/StreamMapper.java

```diff
@@ -302,7 +306,7 @@ public void close() throws InterruptedException {
     }
 
     /** called on process exit */
-    public void taskComplete() {
+    public void taskComplete(boolean interrupted) {
```

Comment:
the map feeder is only ever interrupted on the stream mapper's say-so. There is no need to pass a flag from stream mapper to the map feeder via interrupting the thread and then pass that flag back to stream mapper from map feeder by adding it this signature.